File: Tarea_8/src/Usuario.java
import java.util.*;
public class Usuario {
    int id;
    String nombre;
    List<Libro> librosRentados;

    public Usuario(String nombre) {
        this.id = new Random().nextInt(9999) + 1; // Genera un número aleatorio entre 1 y 9999
        this.nombre = nombre;
        this.librosRentados = new ArrayList<>();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public java.lang.String getNombre() {
        return nombre;
    }

    public void setNombre(java.lang.String nombre) {
        this.nombre = nombre;
    }

    public java.util.List<Libro> getLibrosRentados() {
        return librosRentados;
    }

    public void setLibrosRentados(java.util.List<Libro> librosRentados) {
        this.librosRentados = librosRentados;
    }
}

File: Tarea_8_PY/Usuario.py
import random

class Usuario:
    def __init__(self, nombre):
        self.id = random.randint(1, 9999)  # Genera un número aleatorio entre 1 y 9999
        self.nombre = nombre
        self.libros_rentados = []

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_nombre(self):
        return self.nombre

    def set_nombre(self, nombre):
        self.nombre = nombre

    def get_libros_rentados(self):
        return self.libros_rentados

    def set_libros_rentados(self, libros_rentados):
        self.libros_rentados = libros_rentados

    def __str__(self):
        return f"Usuario: ID: {self.id}, Nombre: {self.nombre}, Libros Rentados: {len(self.libros_rentados)}"